blocks/v2-tabbed-carousel/v2-tabbed-carousel.css

```diff
@@ -0,0 +1,280 @@
+/* stylelint-disable selector-class-pattern */
+:root {
+  --tabbed-carousel-img-width: 60%;
+}
+
+@keyframes truck-entry {
+  0% {
+    transform: translateX(100%);
+    opacity: 0;
+  }
+
+  25% {
+    opacity: 1;
+  }
+
+  100% {
+    transform: translateX(0);
+  }
+}
+
+/* Full width block */
+main .section.v2-tabbed-carousel-container .v2-tabbed-carousel-wrapper {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+
+/* End Full width block */
+
+.v2-tabbed-carousel__images-container,
+.v2-tabbed-carousel__description-container {
+  display: flex;
+  flex-flow: row nowrap;
+  padding-left: 0;
+  margin: 0;
+}
+
+.v2-tabbed-carousel__description-container {
+  order: 2;
+  overflow: hidden;
+}
+
+.v2-tabbed-carousel__images-container {
+  align-items: flex-end;
+  overflow: scroll hidden;
+  scroll-behavior: smooth;
+  scroll-snap-type: x mandatory;
+  scrollbar-width: none;
+}
+
+.v2-tabbed-carousel__images-container::-webkit-scrollbar {
+  display: none;
+}
+
+.v2-tabbed-carousel__image-item {
+  flex: none;
+  scroll-snap-align: center;
+  text-align: center;
+  width: var(--tabbed-carousel-img-width);
+}
+
+.v2-tabbed-carousel__image-item picture {
+  display: block;
+  animation-duration: 1s;
+  animation-delay: 0.5s;
+  animation-name: truck-entry;
+  animation-timing-function: ease-in;
+}
+
+.v2-tabbed-carousel__image-item:first-child {
+  margin-left: calc(100% - var(--tabbed-carousel-img-width)/2);
+}
+
+.v2-tabbed-carousel__images-container::after {
+  content: '';
+  display: block;
+  flex: 0 0 calc(100% - var(--tabbed-carousel-img-width)/2);
+}
+
+.v2-tabbed-carousel__image-item img {
+  width: 100%;
+  object-fit: contain;
+  max-height: 570px;
+}
+
+.v2-tabbed-carousel__content {
+  width: 90%;
+  margin: 0 auto 32px;
+  text-align: center;
+}
+
+.v2-tabbed-carousel__desc-item {
+  color: var(--text-color);
+  flex: none;
+  opacity: 0;
+  width: 100%;
+  transition: opacity var(--duration-small) var(--easing-exit),
+    transform 1s var(--easing-entrance);
+  transform: translateX(-50px);
+}
+
+.v2-tabbed-carousel__desc-item.active {
+  opacity: 1;
+  transform: translateX(0);
+  transition: opacity var(--duration-large) var(--easing-entrance),
+    transform 1s var(--easing-entrance);
+}
+
+.v2-tabbed-carousel__text {
+  text-wrap: balance;
+}
+
+.v2-tabbed-carousel__title {
+  margin-right: 5px;
+  transform: translateY(1px);
+}
+
+.v2-tabbed-carousel__title,
+p.v2-tabbed-carousel__description {
+  display: inline;
+  color: var(--c-grey-800);
+  font-size: var(--f-heading-5-font-size);
+  line-height: var(--f-heading-5-line-height);
+}
+
+.v2-tabbed-carousel__buttons-container {
+  margin-top: 32px;
+}
+
+.v2-tabbed-carousel__buttons-container .button-container {
+  margin: 10px 0;
+}
+
+/* Navigation */
+ul.v2-tabbed-carousel__navigation {
+  display: flex;
+  flex-flow: row nowrap;
+  overflow: auto;
+  list-style: none;
+  margin: 35px 0;
+  padding: 0 35px;
```

Comment:
Changing this to `padding: 2px 35px` would make the button focus state properly visible